docsteady/utils.py

```diff
@@ -170,7 +172,8 @@ def download_and_rewrite_images(value):
         im = Image.open(fs_path)
         width, height = im.size
         if width > Config.MAX_IMG_PIXELS:
-            print(f"[WARNING] Image {fs_path} width greater than {Config.MAX_IMG_PIXELS}")
+            print(f"[WARNING] Image {fs_path} width greater than {Config.MAX_IMG_PIXELS} pixels.")
+            img["width"] = f"{Config.MAX_IMG_PIXELS}px"
```

Comment:
If this is working now, that's great. Specifying the image width is where I was getting into trouble.

...but maybe that's moot if we have a warning somewhere people can find.